### DISCUSSION TITLE
Quick setup — if you’ve done this kind of thing before

### DISCUSSION THREAD

**Discussion by DJSandsCreation** [community/community#23743] [New to GitHub]
<p>So I used to have this page pop up … .</p>
<p><strong>Quick setup</strong>  — if you’ve done this kind of thing before</p>
<p>and I hadn’t been using it however now I am wanting to push a repository from the command line and I am not seeing that http:// <a href="http://github.com/" rel="nofollow noopener">github.com/</a>. . . . .<br>
bit, did I turn it off ? Can I turn it back on??</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! Welcome back :slight_smile:</p>
<p>I’m pretty sure that drop down can’t be turned off.</p>
<p>This document might help you if you’re pushing a new repository to GitHub using the command line:</p>
<p><a href="https://help.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line" class="onebox" target="_blank">https://help.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line</a></p>
<p>And if you want to use GitHub Desktop:</p>
<p><a href="https://help.github.com/en/desktop/contributing-to-projects/adding-a-repository-from-your-local-computer-to-github-desktop" class="onebox" target="_blank">https://help.github.com/en/desktop/contributing-to-projects/adding-a-repository-from-your-local-computer-to-github-desktop</a></p>
